sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,15 +1,20 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
   <Enumeration>SummaryState_DisabledState,SummaryState_EnabledState,SummaryState_FaultState,SummaryState_OfflineState,SummaryState_StandbyState</Enumeration>
-  <Enumeration>DetailedState_DisabledState,DetailedState_EnabledState,DetailedState_FaultState,DetailedState_OfflineState,DetailedState_StandbyState</Enumeration>
+  <Enumeration>DisabledState_DisabledState,DisabledState_EnabledState,DisabledState_FaultState,DisabledState_OfflineState,DisabledState_StandbyState</Enumeration>
+  <Enumeration>ChillerId_chiller01=1,ChillerId_chiller02=2,ChillerId_chiller03=3</Enumeration>
+  <Enumeration>CrackId_crack01=1,CrackId_crack02=2</Enumeration>
+  <Enumeration>FancoilId_fancoil01=1,FancoilId_fancoil02=2,FancoilId_fancoil03=3,FancoilId_fancoil04=4,FancoilId_fancoil05=5,FancoilId_fancoil06=6,FancoilId_fancoil07=7,FancoilId_fancoil08=8,FancoilId_fancoil09=9,FancoilId_fancoil10=10,FancoilId_fancoil11=11,FancoilId_fancoil12=12</Enumeration>
+  <Enumeration>ManejadoraId_manejadoraSblanca=1,ManejadoraId_manejadoraSlimpia=2</Enumeration>
+  <Enumeration>ManejadoraLowerId_manejadoraLower01=1,ManejadoraLowerId_manejadoraLower02=2,ManejadoraLowerId_manejadoraLower03=3,ManejadoraLowerId_manejadoraLower04=4</Enumeration>
```

Comment:
Please consider having a unique ID value for each device. The big advantage is type safety: it is impossible to use the wrong bitmask and get surprised by which device is configured -- instead it'll reference an invalid device. (C++ and Python can enforce type safety without doing this, but SAL transmits these values as an integer).

I see two basic approaches to this:
* Use the index of the bit in the enable/disable mask as the ID. Thus the ID values and enable/disable mask have an obvious 1:1 relation and you could even compute one from the other in ts_idl. With my example just above you'd have chiller01=1, chiller02=2, chiller03=3, fancoil01=4, ... fancoil12=15, ...
* Use the enable/disable bit values "as is". This gives you exactly one enum class and exactly one way to identify a device. This is clearly the best approach if you think you ever might want to support one configuration command configuring multiple devices. Otherwise...I have mixed feelings.